### PR TITLE
[DOCS-5455] Aspose connector

### PR DESCRIPTION
**Pull request by m-hulbert**

### CODE REVIEW

File: docs/modeling/connectors/ootb/aspose.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+Title: Aspose connector
+---
+
+# Aspose connector
+The Aspose connector is used to generate `docx` and `pdf` files using an Aspose template and metadata map. The generated document is saved to the Alfresco Content Services repository folder that corresponds to the process instance ID that called the connector.
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/aspose.md]: Just a clarification, the complete path is:

Application Site / Document Library / Process Folder / Service Task Folder / Generated Document

File: docs/modeling/connectors/ootb/aspose.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Input parameters 
+The following are the parameters that can be passed to the Aspose connector as input parameters using the `GENERATE` action:
+
+| Parameter | Description | Type | Required? |
+| --------  | ----------- | ---- | --------- |
+| `nodeId` | The node ID of the template to use from Alfresco Content Services | String | `*` |
+| `uri` | The URI of the template to use | String | `*` |
+| `files` | A template [file](../../modeling/files.md) uploaded in a process and set as a process variable | File | `*` |
+| `metadata` | The metadata to use in the template | JSON | No |
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/aspose.md]: If no metadata is passed to the connector, the inbound variable map will be used as metadata.

File: docs/modeling/connectors/ootb/aspose.md (hunk 1 above)
Comment (m-hulbert) [reply docs/modeling/connectors/ootb/aspose.md]: I've included a path to the document and assumed it is <service-task-id> that's used. Thanks for the info, I think it's useful to know that!

File: docs/modeling/connectors/ootb/aspose.md (hunk 2 above)
Comment (m-hulbert) [reply docs/modeling/connectors/ootb/aspose.md]: Updated the description to note this.

**Review by mmartinadanx — APPROVED** [review]

**Pull request merged**